tools/bin/commands/minishift

```diff
@@ -94,13 +94,13 @@ minishift::run() {
         # Adapt
         add_user_permissions_for_operator "$user" "true"
 
-        # Relogin with original user
-        $revert_login
-
         # Deploy operator and wait until its up
         result=$(deploy_syndesis_operator)
         check_error "$result"
 
+        # Relogin with original user
+        $revert_login
+
```

Comment:
@zregvart  Actually we allowed that the operator can be installed by any use (if the CRD is set up). This is a new requirement which I'm not sure that I like as now only cluster-admin can install Syndesis, which is a severe constraint (e.g. that would mean that even when the CRD is set up probably a provisioning script on OSO would have to run with full admin permissions to deploy Syndesis (when we switched to CRs).

Actually *why* do we need ClusterRoles for the OAuth proxy ? I have strong feelings that this should be not the case.

Reply:
> Actually why do we need ClusterRoles for the OAuth proxy ? I have strong feelings that this should be not the case.

We do not need any additional privileges for the `oauth-proxy`, we need additional privilege for the `syndesis-operator` service account to be able to create `OAuthClient` resource used by the `oauth-proxy`.